Sources/ProjectDescription/Target.swift

```diff
@@ -53,6 +53,8 @@ public struct Target: Codable, Equatable {
     /// Environment variables to be exposed to the target.
     public let environment: [String: String]
 
+    public let launch: [String: Bool]
```

Comment:
What about calling this one `launchArguments` and making it `[String]` instead of `[String: Bool]`? If developers want to set launch arguments where some of them are disabled they could use custom schemes.
Moreover, I'd add documentation comment. That's important for developers to get inline documentation when editing their manifests.

Reply:
I don't agree on not allowing enabling or disabling the launch argument.

When introducing a launch argument, you as the maintainer know wether or not the launch agument should be enabled or is only presented as a courtesy to the xcode user.

The use case we do this is to allow certain things to be enabled during development, but toggle them off when not launched from xcode.

Another usecase we have is for example a json logging flag, a developer will e able that IF he wants to dump json to the console.

Both are valid, but both indicate a diferent default state requirement for the launch argument.